Titanium.Web.Proxy/Network/CertificateManager.cs

```diff
@@ -219,6 +220,51 @@ public void TrustRootCertificate()
         }
 
         /// <summary>
+        /// Puts the certificate to the local machine's certificate store. 
+        /// Needs elevated permission. Works only on Windows.
+        /// </summary>
+        /// <returns></returns>
+        public bool TrustRootCertificateAsAdministrator()
+        {
+            if (RunTime.IsRunningOnMono())
+            {
+                return false;
+            }
+
+            var fileName = Path.GetTempFileName();
+            File.WriteAllBytes(fileName, RootCertificate.Export(X509ContentType.Pkcs12));
+
+            var info = new ProcessStartInfo
+            {
+                FileName = "certutil.exe",
+                Arguments = "-importPFX -Enterprise -p \"\" -f \"" + fileName + "\"",
```

Comment:
I don't like the idea of embedding at least for now. We can add a link to the exe (may be separate repository) in our readme & wiki, and if it got considerable use we can import it to this repo.

Having said that I am not sure how an exe would work in Linux or Mac since they have different extensions for executable if I recall correctly.